Experiments_toolkit.py

```diff
@@ -76,35 +76,32 @@ def preprocess_data(DATA, rows, filtering_params, window_size, offset):
                                 these windows (numpy arrays) into a dictionary.
 
        Args:
-           DATA -- dict (Dictionary of keys and within the keys are the 3D data samples)
-           rows -- int (Number of rows in the data)
-           filtering_params -- All_Filtering_Params (Dataclass found in windowing.py that 
+        	DATA -- np array (Numpy array where the last row is time steps)
+            rows -- int (Number of rows in the data)
+            filtering_params -- All_Filtering_Params (Dataclass found in windowing.py that 
                                                         stores important parameters for the 2 steps involved with 
                                                         finding the precise start and end time of a lift in the data)
             window_size -- int (Window size of the trace to be saved into the dictionary that will be
                                                 returned)
-                offset -- int (Number to add to the predicted start time of the lift from where the window
+            offset -- int (Number to add to the predicted start time of the lift from where the window
                                                 will start at)
 
        Return:
            windowed_data_dictionary -- dict (Dictionary that stores the windows under the "windows" key)
            """
     windowed_data_dictionary = {"windows": []}
```

Comment:
what's the reason for putting the output into a dict?

Reply:
this change must not have made it into the commit I pushed but this function now returns a int[]

Reply:
no sorry it returns a numpy array